spec.html

```diff
@@ -44478,15 +44481,21 @@ <h1>Properties of DisposableStack Instances</h1>
               [[DisposableState]]
             </td>
             <td>
-              One of ~pending~ or ~disposed~. Governs how a disposable stack will react to incoming calls to its `@@dispose` method.
+              ~pending~ or ~disposed~
+            </td>
+            <td>
+              Governs how a disposable stack will react to incoming calls to its `@@dispose` method.
             </td>
           </tr>
           <tr>
             <td>
               [[DisposableResourceStack]]
             </td>
             <td>
-              A List of DisposableResource Records.
+              a List of DisposableResource Records
+            </td>
+            <td>
+              ?
```

Comment:
```suggestion
              An ordered list of resources to be disposed when the disposable stack is disposed.
```

Reply:
Lists are ordered by definition, so you could drop "ordered". And "a list of" isn't really necessary, given the type right there. So maybe just:
```suggestion
              Resources to be disposed when the disposable stack is disposed.
```

Reply:
That works

Reply:
I can't seem to commit the above suggestion, so I'll merge this and make the change in a separate commit.